Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelPage.xaml.cs

```diff
@@ -67,6 +71,22 @@ public APIModelPage(ApplicationAPIModel applicationAPIModelBase)
             UpdateOutputTemplateTabHeader();
         }
 
+        private void DeleteParamEvent(ParamEventArgs eventArgs)
+        {
+            bool someParamContainPath = false;
+            foreach (AppModelParameter param in eventArgs.ModelParamsList)
+                if (!string.IsNullOrEmpty(param.Path))
+                    someParamContainPath = true;
+
+
+            APIModelBodyNodeSyncPage bodyNodeSyncPage;
+            if (someParamContainPath)
+            {
+                bodyNodeSyncPage = new APIModelBodyNodeSyncPage(mApplicationAPIModel, eventArgs.ModelParamsList);
+                bodyNodeSyncPage.ShowAsWindow();
+            }
+        }
+
```

Comment:
please explain this DeleteParamEvent, why we need the parent API page to handle it? why it can't be done inside the ModelParamsPage?

Reply:
This is how you told me to do it in the start,
in AppModelPage we add the APIModel itself and give it to APIModelBodyNodeSyncPage